### DISCUSSION TITLE
Welcome to v2ray-core Discussions!

### DISCUSSION THREAD

**Discussion by vcptr** [v2fly/v2ray-core#496] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicholascw** [comment]
好 耶

**Comment by CalmLong** [comment]
前排围观

**Comment by MuenYu** [comment]
芜湖~

**Comment by cbjn** [comment]
haha
